packages/confirmation-modal/src/ConfirmationModal.spec.tsx

```diff
@@ -146,4 +146,16 @@ describe('packages/confirmation-modal', () => {
       expect(confirmationButton).toBeDisabled();
     });
   });
+
+  describe('submit is disabled when', () => {
+    test('"submitDisabled" prop is set', () => {
```

Comment:
would be also good to test this in combination with the `requiredInputText` prop

Reply:
👍